brayns/common/types.h

```diff
@@ -298,13 +298,13 @@ enum MorphologyLayoutType
     ML_GRID,
 };
 
-enum CameraType
+enum class CameraType
```

Comment:
Not in this case, it has to be consistent with the rest of the application code.